docs/manual/templates_system/articles.rst

```diff
@@ -0,0 +1,49 @@
+Handling Articles
+=================
+
+Listing Articles
+----------------
+
+Publisher have concept of Meta Loaders - one of built in loaders covers articles.
+
+The :code:`articles` loader parameters:
+
+ * (optional) key :code:`route` - id or name or array of id's used for loading meta (if omitted then current route is used).
+
+.. code-block:: twig
+
+    {% gimmelist articles from articles %} <!-- It will use route from context -->
```

Comment:
it should be `{% gimmelist article from articles %}` instead of `{% gimmelist articles from articles %}`. See also lines below in that file.